tests/test_notation_sets.py

```
@@ -0,0 +1,182 @@
+import pytest
```

Pylint (4.0.7)

[error] Unable to import 'pytest'

(E0401)